domain/src/main/java/org/oppia/android/domain/topic/TopicListController.kt

```diff
@@ -453,17 +462,19 @@ class TopicListController @Inject constructor(
     // The comments describe the correct dependencies, but those might not be available until the
     // topic is introduced into the app.
     return when (topicId) {
-      // TEST_TOPIC_ID_0 (depends on Fractions)
-      TEST_TOPIC_ID_0 -> listOf(FRACTIONS_TOPIC_ID)
-      // TEST_TOPIC_ID_1 (depends on TEST_TOPIC_ID_0,Ratios)
-      TEST_TOPIC_ID_1 -> listOf(TEST_TOPIC_ID_0, RATIOS_TOPIC_ID)
+      // Place values depends on nothing since it's the first set of lessons to play.
+      REAL_PLACE_VALUES_TOPIC_ID -> listOf()
       // Fractions (depends on A+S, Multiplication, Division)
-      FRACTIONS_TOPIC_ID -> listOf()
+      REAL_FRACTIONS_TOPIC_ID -> listOf(REAL_MULTIPLICATION_TOPIC_ID)
       // Ratios (depends on A+S, Multiplication, Division)
-      RATIOS_TOPIC_ID -> listOf()
+      REAL_RATIOS_TOPIC_ID -> listOf(REAL_MULTIPLICATION_TOPIC_ID)
       // Addition and Subtraction (depends on Place Values)
-      // Multiplication (depends on Addition and Subtraction)
+      REAL_ADDITION_AND_SUBTRACTION_TOPIC_ID -> listOf(REAL_PLACE_VALUES_TOPIC_ID)
+      // Multiplication (depends on Addition and Subtraction); depend instead on Place Values as a
+      // poor proxy for A+S since that topic isn't yet available in the app.
+      REAL_MULTIPLICATION_TOPIC_ID -> listOf(REAL_PLACE_VALUES_TOPIC_ID)
```

Comment:
I am unable to understand the above comment but don't we need `REAL_ADDITION_AND_SUBTRACTION_TOPIC_ID ` as prerequisite for `REAL_MULTIPLICATION_TOPIC_ID `

Reply:
No--addition and subtraction isn't actually included yet because we can't support it in the app. It's listed as as an upcoming topic (& will show up when all other topics are finished). Since upcoming topics can't be finished, making it a prerequisite here will break the graph/make it impossible to recommend multiplication.